client/lib/domains/constants.js

```diff
@@ -17,7 +17,40 @@ const registrar = {
 	MAINTENANCE: 'Registrar TLD Maintenance'
 };
 
+const tlds = {
```

Comment:
@yurynix Why is it less fragile to match the product slugs with a regex than to just store the slugs themselves on the client like we do with other products? Instead of an explicit convention that is shared between the client/server (i.e. that .com domains use `domain_reg` as their product slug), we are creating an implicit convention (i.e. that `product_name` must match the regex above).

These values are hardly ever going to change, so I don't think there is much of a maintenance cost by adding them to the client.